spec.html

```diff
@@ -16621,15 +16624,14 @@ <h1>Runtime Semantics: Evaluation</h1>
           <h1>HostGetImportMetaProperties ( _moduleRecord_ )</h1>
           <p>The host-defined abstract operation HostGetImportMetaProperties takes argument _moduleRecord_ (a Module Record). It allows hosts to provide property keys and values for the object returned from `import.meta`.</p>
 
-          <p>The implementation of HostGetImportMetaProperties must conform to the following requirements:</p>
+          <p>An implementation of HostGetImportMetaProperties must conform to the following requirements:</p>
           <ul>
-            <li>It must return a List, whose values are all Records with two fields, [[Key]] and [[Value]].</li>
+            <li>It must return a normal completion of a List whose values are all Records with two fields, [[Key]] and [[Value]].</li>
             <li>Each such Record's [[Key]] field must be a property key, i.e., IsPropertyKey must return *true* when applied to it.</li>
             <li>Each such Record's [[Value]] field must be an ECMAScript value.</li>
-            <li>It must always complete normally (i.e., not return an abrupt completion).</li>
           </ul>
 
-          <p>The default implementation of HostGetImportMetaProperties is to return a new empty List.</p>
+          <p>The default implementation of HostGetImportMetaProperties is to return a normal completion of a new empty List.</p>
```

Comment:
There doesn't appear to be precedence for the construction "a normal completion of __x__". There are some uses of "a normal completion with a value of __x__", though "NormalCompletion(x)" is more prevalent and more precise. What do you think about using one of those two?

Reply:
I'd like to use the NormalCompletion(x) form, but thought NormalCompletion(a new empty List) would look weird. But now that I've typed that out it's probably fine.

Reply:
What about `NormalCompletion(« »)`?

Reply:
> What about NormalCompletion(« »)

Great idea, will use that.